DEV-4/checkSeq/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace checkSeq
+{
+    class checkSequence
+    {
+        static void Main(string[] args)
+        {
+            bool Checking = true;
+            while (Checking)
+            {
+                try
+                {
+                    Console.Write("Size of your sequence is n = ");
+                    String sizeArray = Console.ReadLine();
```

Comment:
Do not enter size of possible sequences, too easy. The sequences has to be entered as strings delimited by spaces, it was discussed in skype